swagger/swagger.yaml

```diff
@@ -0,0 +1,710 @@
+        schema:
+          $ref: "#/definitions/QuestionIn"
+      responses:
+        200:
+          description: "Question updated"
+          schema:
+            $ref: "#/definitions/Question"
+        404:
+          description: "Question not found"
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+        400:
+          description: "Invalid input"
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+    delete:
+      tags:
+      - "leader"
+      summary: "Delete question"
+      description: "Delete a question from the system."
+      operationId: "deleteQuestion"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "path"
+        name: "churchUuid"
+        type: "string"
+        format: "uuid"
+        required: true
+      - in: "path"
+        name: "questionUuid"
+        type: "string"
+        format: "uuid"
+        required: true
+      responses:
+        204:
+          description: "Question deleted"
+        404:
+          description: "Question not found"
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+
+  /church/{churchUuid}/matchgroup/{matchGroupUuid}:
+    get:
+      tags:
+      - "member"
+      summary: "Retrieve group"
+      description: "Retrieve a group of matched members for the given UUIDs."
+      operationId: "getMatchGroup"
+      parameters:
+      - in: "path"
+        name: "churchUuid"
+        type: "string"
+        format: "uuid"
+        required: true
+      - in: "path"
+        name: "matchGroupUuid"
+        type: "string"
+        format: "uuid"
+        required: true
+      responses:
+        200:
+          description: "Group of matched church members"
+          schema:
+            $ref: "#/definitions/Group"
+        404:
+          description: "Group not found"
+          schema:
+            $ref:  "#/definitions/ErrorResponse"
+
+definitions:
+  UserIn:
+    type: "object"
+    properties:
+      email:
+        type: "string"
+        format: "email"
+      firstName:
+        type: "string"
+      lastName:
+        type: "string"
+      contact:
+        type: "array"
+        items:
+          $ref: "#/definitions/ContactMethodIn"
+      dateOfBirth:
+        type: "string"
+        format: "date"
+      description:
+        type: "string"
+
+  User:
+    type: "object"
+    properties:
+      uuid:
+        type: "string"
+        format: "uuid"
+      email:
+        type: "string"
+        format: "email"
+      firstName:
+        type: "string"
+      lastName:
+        type: "string"
+      contact:
+        type: "array"
+        items:
+          $ref: "#/definitions/ContactMethod"
+      gender:
+        type: "string"
+        enum:
+          - "male"
+          - "female"
+      dateOfBirth:
+        type: "string"
+        format: "date"
+      description:
+        type: "string"
+      church:
+        type: "string"
+        format: "uuid"
+
+  PublicUser:
```

Comment:
Is the only difference between `PublicUser` and `User` the DOB? (For privacy issues, I'm assuming?)